dwitter/templates/dweet/dweet.html

```diff
@@ -39,6 +47,38 @@
           case "hideStats":
             setStatsVisibility(false);
             break;
+          case "record":
+            recording = true;
+            encoder = new GIF({ 
+              workers: 4, 
```

Comment:
Could be worthwhile to change this depending on hardware? I'm not sure how well supported it is; but found this: https://stackoverflow.com/questions/3289465/get-number-of-cpu-cores-in-javascript?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa 

Should have a default fallback if the API isn't available.

Not critical, so I'm happy to merge this with `workers: 4` and open an issue instead